### ISSUE TITLE
[ADDITION]: Addition of Templates 

### ISSUE THREAD

**Issue by AbhijithGanesh** [dokc/Helm-Charts#15]
We need templates for Issues, features and bug-reports which are written as YAML files, you can refer them here

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

**Comment by ManasviGoyal** [comment]
Can I work on this?

**Comment by AbhijithGanesh** [comment]
Assignedv@ManasviGoyal

**Comment by AbhijithGanesh** [comment]
Would you like to split the work for this task or would you like to do all of them @ManasviGoyal ?

**Comment by ManasviGoyal** [comment]
@AbhijithGanesh Yes. Sure I can split the work. I will be working on 1 and 2. so, if any one wants? they can take up the rest.
